locale/en/docs/guides/getting-started-guide.md

```diff
@@ -25,4 +25,8 @@ server.listen(port, hostname, () => {
 });
 ```
 
-After that, run your web server using `node app.js`, visit `http://localhost:3000`, and you will see a message 'Hello World'
+Now, run your web server using `node app.js`. Visit `http://localhost:3000` and
+you will see a message saying "Hello World".
+
+Refer to the [Introduction to Node.js](https://nodejs.dev/) for a more
```

Comment:
Is nodejs.dev officially launched yet?

Reply:
> Is nodejs.dev officially launched yet?

Yes, as of one year and two days ago.

* https://medium.com/@nodejs/node-js-selected-by-google-for-dev-top-level-domain-early-access-6b0c5f9205f8
* https://twitter.com/nodejs/status/1097903920242937858